variables.tf

```diff
@@ -201,3 +201,22 @@ variable "putin_khuylo" {
   type        = bool
   default     = true
 }
+
+variable "intelligent_tiering_config" {
+  description = <<EOS
```

Comment:
Please make the description as a string (not heredoc), no link to the documentation, and no comments for the type. See line 146, for example.